test/check/check.go

```diff
@@ -73,6 +73,11 @@ func CheckComponentsRemoval(components []Component) {
 			return checkForComponentRemoval(&component)
 		}, 5*time.Minute, time.Second).ShouldNot(HaveOccurred(), fmt.Sprintf("%s component has not been fully removed within the given timeout", component.ComponentName))
 	}
+
+	By(fmt.Sprintf("Checking the %s configmap has been removed", names.AppliedPrefix+names.OperatorConfig))
```

Comment:
<aybe worth to have its own function please ?
it isn't a component

Potential FUP (follow-up) to think about
maybe we can generalize `CheckComponentsRemoval`, that it scan according ownerRef / relation labels what should be deleted ? (not crucial atm i guess)

Reply:
Let's think about that in  a follow up